content/cli/commands/produce.md

```diff
@@ -62,6 +62,26 @@ Ok!
 
 The producer splits the key from the value and prints it in a `[key] value` format.
 
+### Produce using a SmartModule
```

Comment:
Question: Is `--smartmodule-path` the only way to produce with smartmodules? This is great for testing, but not ideal when multiple clients want to call the same smartmodule when producing. Is there also a command for the producer to use a smartmodule that has already been uploaded?

Reply:
That's great! It may be best to add a 2nd paragraph, that way we cover both cases.